ic.nix

```diff
@@ -106,6 +107,11 @@ let
         "-Lall=${libiconv-static.out}/lib"
         "-lstatic=iconv"
       ];
+      RUST_SRC_PATH = "${
+          pkgs.rust-bin.stable.latest.default.override {
+            extensions = [ "rust-src" ];
```

Comment:
How about adding the extension to the `rust-stable` definition in `default.nix`? Then in this file, just use `${rust-stable}/lib/rustlib/src/rust/library`.